spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -238,6 +242,11 @@ class Analyzer(tableUtils: TableUtils,
            |""".stripMargin)
     }
 
+    if (validationAssert) {
+      assert(checkTs == "Acceptable Timestamp Input",
+             "[ERROR]: GroupBy validation failed. Please check that source has valid timestamps.")
+    }
```

Comment:
As mentioned, we can remove the if condition to check it always.

Reply:
Done.